### PR TITLE
WC: Snackbar

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
Added snackbar component

### CODE REVIEW

File: Documentation/Components/Snackbar/snackbar.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Snackbar</title>
+    <link rel="stylesheet" href="snackbar.css">
+</head>
+
+<body>
+    <div class="snackbar flex-row">
+        <p class="para-s">Can't send picture. Retry in 5 seconds.</p>
+        <button class="snackbar-btn retry-btn">RETRY</button>
```
Comment (devananth) [inline Documentation/Components/Snackbar/snackbar.html]: Haven't used `retry-btn` class in css , if it's not necessary. Remove it

**Review by devananth — CHANGES_REQUESTED** [review]
Clean code.

File: Documentation/Components/Snackbar/snackbar.html (hunk 1 above)
Comment (Prabhav-Sharma) [reply Documentation/Components/Snackbar/snackbar.html]: Oh, I forgot to remove it, thank you!

**Review by devananth — APPROVED** [review]
you can merge it !

**Review by barbaricankit — APPROVED** [review]
Looks good.

**Pull request merged**